modmesh/gui/naca.py

```diff
@@ -112,10 +149,11 @@ def runmain():
     A simple example for drawing a couple of cubic Bezier curves.
     """
     w = core.WorldFp64()
-    draw_naca4(w, number='0012', npoint=101, fac=5.0, off_x=0.0, off_y=2.0,
-               open_trailing_edge=False, cosine_spacing=True)
+    draw_naca4(w, number="0012", npoint=101, fac=5.0, off_x=0.0, off_y=2.0,
```

Comment:
It is not necessary to replace the single quote `'` with double quote `"`.  Please revert the unnecessary change.